### PR TITLE
PS-8481: Add OL9 and ubuntu-jammy in InnoDB cluster Job

### PR DESCRIPTION
**Pull request by kaushikpuneet07**

### CODE REVIEW

File: ps/jenkins/test-ps-innodb-cluster.groovy [hunk 1]
```diff
@@ -1,6 +1,6 @@
 library changelog: false, identifier: 'lib@master', retriever: modernSCM([
     $class: 'GitSCMSource',
-    remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
+    remote: 'https://github.com/kaushikpuneet07/jenkins-pipelines.git'
```
Comment (eleo007) [inline ps/jenkins/test-ps-innodb-cluster.groovy:3]: revert

File: ps/jenkins/test-ps-innodb-cluster.groovy [hunk 2]
```diff
@@ -16,7 +16,7 @@ void installDependencies() {
 
     sh '''
         rm -rf package-testing
-        git clone https://github.com/Percona-QA/package-testing
+        git clone -b PS-8481 https://github.com/kaushikpuneet07/package-testing
```
Comment (eleo007) [inline ps/jenkins/test-ps-innodb-cluster.groovy:19]: revert

File: ps/jenkins/test-ps-innodb-cluster.yml [hunk 3]
```diff
@@ -14,9 +14,9 @@
     pipeline-scm:
         scm:
             - git:
-                url: https://github.com/Percona-Lab/jenkins-pipelines.git
+                url: https://github.com/kaushikpuneet07/jenkins-pipelines.git
```
Comment (eleo007) [inline ps/jenkins/test-ps-innodb-cluster.yml:17]: revert

File: ps/jenkins/test-ps-innodb-cluster.yml [hunk 4]
```diff
@@ -14,9 +14,9 @@
...
                 branches:
-                    - 'master'
+                    - 'innodb-job'
```
Comment (eleo007) [inline ps/jenkins/test-ps-innodb-cluster.yml:19]: revert

**Review by eleo007 — CHANGES_REQUESTED** [review]

File: ps/jenkins/test-ps-innodb-cluster.groovy (hunk 1 above)
Comment (kaushikpuneet07) [reply ps/jenkins/test-ps-innodb-cluster.groovy]: Fixed

File: ps/jenkins/test-ps-innodb-cluster.groovy (hunk 2 above)
Comment (kaushikpuneet07) [reply ps/jenkins/test-ps-innodb-cluster.groovy]: Fixed

File: ps/jenkins/test-ps-innodb-cluster.yml (hunk 3 above)
Comment (kaushikpuneet07) [reply ps/jenkins/test-ps-innodb-cluster.yml]: Fixed

File: ps/jenkins/test-ps-innodb-cluster.yml (hunk 4 above)
Comment (kaushikpuneet07) [reply ps/jenkins/test-ps-innodb-cluster.yml]: Fixed

**Review by eleo007 — APPROVED** [review]

**Pull request merged**